app/javascript/src/Types/colorbox.d.ts

```diff
@@ -0,0 +1,293 @@
+/* eslint-disable */
+
+/**
+ * This was copied from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jquery.colorbox/index.d.ts
+ * but colorbox is injected into rails' jquery, which is different from the one we import from
+ * node_modules, so we need to reference JQueryStaticV1
+ */
```

Comment:
as a noob I can't understand this comment. So what is this thing below overriding? Or does it do something else?

Reply:
Originally this file, when imported in a Typescript file, overrides `JQueryStatic` which is the jquery imported from `node_modules` (v3.x). However, we inject `colorbox` into rails' jquery (v1.x) so I changed this file `colorbox.d.ts` to reflect that. Our version do not override `JQueryStatic` and instead is imported manually here: https://github.com/3scale/porta/pull/3700/files#diff-2b63f112c8f3de42426d6bcd88229ee7a9274ee1104fa6c76bbdf511c84f6190R39